src/index.scss

```diff
@@ -224,7 +224,6 @@ body {
       }
 
       .pause-button {
-        width: 47px;
         text-align: center;
       }
```

Comment:
can you give a fixed width to the pause button? Otherwise it will change dimension when you click on it because the label changes from STOP to RUN and viceversa.